_sass/components/_c-title-popup.scss

```diff
@@ -0,0 +1,59 @@
+.c-title-popup {
```

Comment:
Are you reusing this title out of the pop-up?
If not, it is not necessary yo make a new component.